_config.yml

```diff
@@ -16,6 +16,7 @@
 title: PELUX®
 email: info@pelagicore.com
 description: PELUX is a Linux based platform for your automotive project.
```

Comment:
I know you're not changing this here but I though I comment here before I forget: "your" sounds way too personal and direct to me. Maybe we just say "for IVI systems"?